### DISCUSSION TITLE
local instance can't find address which the public server can.

### DISCUSSION THREAD

**Discussion by roeezamir** [osm-search/Nominatim#2740] [Q&A] [answered]
Hello!

I run local instance (`mediagis/nominatim:4.0` docker image) with tiger data and us postcodes
And from some reason i can't find usa addresses which can be found in the public sever ([nominatim.openstreetmap.org](url))
For example:
The following request returns nothing
`http://{myhostname}:8080/search?street=3+PINE&state=georgia&city=ELLAVILLE&country=united+states&format=json`
But the following request:
`https://nominatim.openstreetmap.org/search?street=3+PINE&state=georgia&city=ELLAVILLE&country=united+states&format=json`

returns osm ids: W9457253, W9456669.

Just to make sure i sent lookup request to make sure these object are exist in my local instance and they exist.
 
it might because of the versions difference?
I tried to use the ICU tokenzier as the below topic hinted to be used by the public server
So only difference is the versions or did i miss something?

(Not sure if it relevant, but i use flat node file)

Thanks in advance

**Comment by mtmail** [comment, marked as answer]
It's like the version difference. nominatim.osm.org runs the latest (master branch) and there have been changes to street name matching. https://github.com/osm-search/Nominatim/pull/2553

I run a Nominatim version from last month, it returns
- `    "display_name": "3, Pine Circle, Ellaville, Schley County, Georgia, 31806, United States",`
- `    "display_name": "3, Pine Drive, Ellaville, Schley County, Georgia, 31806, United States",`

A search for the full name `Pine Circle` or `Pine Drive` (or abbreviated `Pine Dr`) should work on your installation, too.

nominatim.osm.org uses the new ICU tokenizer (https://nominatim.org/2021/11/03/release-40.html) which will be default in the next release. If you use the legacy tokenizer, it's configurable, that might be another difference. Changing the tokenizer requires a full reimport of data but it's worth trying.

**Reply by roeezamir** [reply]
Thans for the very quick check and answer! :)

**Comment by roeezamir** [comment]
Hi again,
Can you please specify the commit id you are using?
10x

**Reply by mtmail** [reply]
8c073993 (May 4th)

**Reply by roeezamir** [reply]
Thanks! :)
